src/sagas.js

```diff
@@ -0,0 +1,78 @@
+import { call, put, takeEvery, takeLatest } from 'redux-saga/effects'
+import {fetchMovies, fetchMovieDetails} from './moviesAPI'
+import { MOVIES_FETCH_SUCCEEDED, SHOW_MOVIE_DETAILS, DETAILS_FETCH_SUCCEEDED, GET_MORE_RESULTS, ERROR } from './redux/actions'
+
+//get movies for search
+function* getMovies(action) {
```

Comment:
Very nice implementations of the Sagas in general in this file.

Please note that in "real life" projects, we usually define a Saga per reducer, something like `userSaga`, `feedSaga`, etc, etc